app/components/blacklight/top_navbar_component.html.erb

```diff
@@ -1,19 +1,17 @@
 <div data-controller="feedback analytics" class="bg-dark">
-  <div class="container">
```

Comment:
Without this container the content of the `div` within in the `nav` expands to the full width of the window. Maybe the thing to do is change the inner div class to `container` instead of `container-fluid` that is getting returned by the `container_classes` helper?

This PR:
<img width="791" alt="Screenshot 2024-06-13 at 8 22 19 AM" src="https://github.com/sul-dlss/stanford-arclight/assets/458247/fc6cdf5a-2d34-4d1a-bf42-6cc3dc227734">

Main:
<img width="791" alt="Screenshot 2024-06-13 at 8 21 00 AM" src="https://github.com/sul-dlss/stanford-arclight/assets/458247/cbea75ae-109e-4b6f-866a-4e7afb5359b0">